docs/questions/cssQuiz.html

```diff
@@ -0,0 +1,118 @@
+                        </ul>
+                    </div>
+                    
+                </div>
+            </header>
+            <section class="section">
+                <div class="container">
+                    <div class="heading">
+                        <h1>Play the css quiz.</h1>
+                    </div>
+                    <div class="quiz-box-container">
+                        <div class="quiz-box">
+                            <div class="quiz-header">
+                                <p>Question 1/5 </p>
+                            </div>
+                            <div class="quiz-question">
+                                <p>Which of the below CSS properties represent the order of flex items in the grid container ?</p>
+                            </div>
+                            <div class="quiz-option-list">
+                                <div class="option">float</div>
+                                <div class="option">order</div>
+                                <div class="option">overflow</div>
+                            </div>
+                        </div>
+                        <div class="quiz-box">
+                            <div class="quiz-header">
+                                <p>Question 2/5 </p>
+                            </div>
+                            <div class="quiz-question">
+                                <p>How do we set the default width of the font in CSS ?</p>
+                            </div>
+                            <div class="quiz-option-list">
+                                <div class="option">font-stretch</div>
+                                <div class="option">text-transform</div>
+                                <div class="option">font-variant</div>
+                            </div>
+                        </div>
+                        <div class="quiz-box">
+                            <div class="quiz-header">
+                                <p>Question 3/5 </p>
+                            </div>
+                            <div class="quiz-question">
+                                <p>Which element is used to represent the transparency of an element in CSS ?</p>
+                            </div>
+                            <div class="quiz-option-list">
+                                <div class="option">overlay</div>
+                                <div class="option">hover</div>
+                                <div class="option">opacity</div>
+                            </div>
+                        </div>
+                        <div class="quiz-box">
+                            <div class="quiz-header">
+                                <p>Question 4/5 </p>
+                            </div>
+                            <div class="quiz-question">
+                                <p>Which of the below is correct syntax when we put a line over text in CSS ?</p>
```

Comment:
Better change the question to line through the text as line over text may get interpreted as line above the text which can be achieved by `text-decoration: overline`

Reply:
sure ...made the changes.